modules/passkey/contracts/SafeWebAuthnSigner.sol

```diff
@@ -22,27 +21,29 @@ contract SafeWebAuthnSigner is SignatureValidator {
     uint256 public immutable Y;
 
     /**
-     * @notice The P-256 verifier used for ECDSA signature validation.
+     * @notice The P-256 verifiers used for ECDSA signature validation.
      */
-    IP256Verifier public immutable VERIFIER;
+    P256.Verifiers public immutable VERIFIERS;
 
     /**
      * @dev Constructor function.
      * @param x The X coordinate of the P-256 public key of the WebAuthn credential.
      * @param y The Y coordinate of the P-256 public key of the WebAuthn credential.
-     * @param verifier The P-256 verifier to use for signature validation. It MUST implement the
-     * same interface as the EIP-7212 precompile.
+     * @param verifiers The P-256 verifiers to use for signature validation. This is the
+     * concatenation of `uint32(precompile) || address(fallback)` specifying the `precompile` to
+     * use as well as the `fallback` Solidity P-256 verifier implementation in case the precompile
```

Comment:
```suggestion
     * use the `fallback` Solidity P-256 verifier implementation in case the precompile
```

Reply:
Hmm, I think it is correct as I wrote it. When I read it out loud it made sense as I wrote it at least.